zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -1348,15 +1348,19 @@ public void processConnectRequest(ServerCnxn cnxn, ByteBuffer incomingBuffer)
         ServerMetrics.getMetrics().CONNECTION_REQUEST_COUNT.add(1);
 
         boolean readOnly = false;
+        boolean logWarning = false;
```

Comment:
With **lpgWarning** I would expect a **true** value as initial value.
With current approach I would name the variable as **warningLogged**

Btw did you mean to declare the variable as a instance field?

Reply:
Hey @eolivelli Thanks for your review. I have changed the variable name as per your suggestion. I didn't mean to declare variable as an instance field but do you think to declare it as an instance field makes more sense?? Please do let me know